libs/libs-pomfix/lib/fix.js

```diff
@@ -0,0 +1,196 @@
+var es = require('event-stream')
+var combine = require('stream-combiner')
+
+var notNull = function (o) { return o !== null }
+var notUndefined = function (o) { return o !== undefined }
+
+var processPom = function ($) {
+  var descriptor = $('plugin descriptor')
+    .map(function () {
+      var descriptor = $(this).text().trim()
```

Comment:
Just use the currentValue argument of the map function, then you won't have to constantly use `this`:
```
.map(function(element){
    var descriptor = element.text().trim()
```
or
```
.map(function(element){
    var match = element.text().trim().match(pattern)
```